assets/components/contribAmounts/contribAmounts.jsx

```diff
@@ -214,10 +216,10 @@ function amountToggles(isoCountry: IsoCountry = 'GB'): AmountToggle {
   };
 }
 
-function contribToggle(isoCountry: IsoCountry = 'GB'): Toggle {
+function contribToggle(isoCountry: IsoCountry = 'GB', showAnnual: boolean): Toggle {
   return {
     name: 'contributions-period-toggle',
-    radios: contribCaptionRadios[isoCountry],
+    radios: showAnnual ? contribCaptionRadios.GB_WITH_ANNUAL : contribCaptionRadios[isoCountry],
```

Comment:
I think you should add the validation of being in GB. Since our AB Test framework doesn't bear in mind countries, I think this can run in the US.

Reply:
I've added a check in where the boolean is originally calculated. 

It would be nice if we could update the AB test framework so that it is aware of country when it works out the participations, I'll put in a card to investigate.

Reply:
Yes, we should add it to the Ab Test framework. I created a card [here](https://trello.com/c/l0fEidly/897-add-country-to-the-abtest-framework)